app/models/manageiq/providers/workflows/automation_manager/workflow_instance.rb

```diff
@@ -87,6 +89,6 @@ def run(args = {})
       object.after_ae_delivery(ae_result)
     end
 
-    run_queue(:zone => zone, :role => role, :object => object) unless wf.end?
+    run_queue(:zone => zone, :role => role, :object => object, :deliver_on => 1.minute.from_now.utc) unless wf.end?
```

Comment:
think I want to expose `wf.waiting`

Reply:
Yeah I was hoping to contain that knowledge inside the `.wait` method